otter/test/integration/test_utils.py

```diff
@@ -0,0 +1,201 @@
+"""
+Tests for the utility functions for convergence black-box testing.
+"""
+from pyrsistent import pmap, pset
+
+from twisted.trial.unittest import SynchronousTestCase
+
+from otter.convergence.model import (
```

Comment:
Given that these are explicitly supposed to be "black-box" tests, I am surprised that internal model classes are being used.